### PR TITLE
Remove help, move to own repo and static site

### PR DESCRIPTION
**Pull request by marten**
Staging branch URL:

Fixes # .

Remove all the PFE project building help pages from PFE. These are now hosted in Markdown format in https://github.com/zooniverse/help/ and rendered to https://zooniverse.github.io/help/

This must not be merged until https://github.com/zooniverse/static/pull/40 is deployed, otherwise these URLs will break.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 FROM node:8.12-alpine
 
+EXPOSE 3735
```
Comment (eatyourgreens) [inline Dockerfile]: Why is this needed, out of interest? I don't have any problem running the app in Docker, in development, and connecting to  localhost:3735.

File: app/pages/lab/project.cjsx [hunk 2]
```diff
@@ -98,13 +98,13 @@ EditProjectPage = createReactClass
             <div className="nav-list-header">Need some help?</div>
             <ul className="nav-list">
               <li>
-                <Link className="nav-list-item" to="/help">Read a tutorial</Link>
+                <a className="nav-list-item" href="https://zooniverse.github.io/help">Read a tutorial</a>
```
Comment (eatyourgreens) [inline app/pages/lab/project.cjsx]: Links from project builder forms should probably use `target=_blank ref="noopener nofollow"` to avoid losing work in progress in the forms while referring to help.

File: app/pages/lab/project.cjsx [hunk 3]
```diff
@@ -98,13 +98,13 @@ EditProjectPage = createReactClass
...
               </li>
               <li>
                 <Link to="/talk/18" className="nav-list-item">Ask for help on talk</Link>
               </li>
               <li>
-                <Link to="/help/glossary" className="nav-list-item">Glossary</Link>
+                <a href="https://zooniverse.github.io/help/getting-started/glossary" className="nav-list-item">Glossary</a>
```
Comment (eatyourgreens) [inline app/pages/lab/project.cjsx]: Ditto re opening a new tab.

**Comment by eatyourgreens** [comment]
Looks good to me. Help links from the project builder should probably open in new tabs, to avoid losing form state while you're in the middle of filling it in.

I'm going to leave this unapproved until zooniverse/static#40 is deployed, but I think this is good to merge.

File: Dockerfile (hunk 1 above)
Comment (marten) [reply Dockerfile]: Removed, can't reproduce why I needed it. I think maybe that HTTP (non-SSL) no longer worked but didn't tell me in any meaningful way, and I discovered both that and tried the EXPOSE at the same time.

**Comment by marten** [comment]
Static is deployed and works well. This can be merged now if @eatyourgreens wants to give it the once over and approval.
